specifyweb/frontend/js_src/lib/components/FormEditor/viewSpec.ts

```diff
@@ -406,11 +406,10 @@ const subViewSpec = (
         syncer(
           (raw: string) => {
             const cellName = cell.rest.node.attributes.name;
-            const cellRelationship = table?.fields
-              .filter((field) => field.isRelationship)
-              .find((table) => table.name === cellName) as
-              | Relationship
-              | undefined;
+            const cellRelationship =
+              typeof cellName === 'string'
+                ? table?.getRelationship(cellName)
```

Comment:
this results in the same error in Sp, table?.getRelationship is not a function

Reply:
Actually, this was caused by a bug in the definition.

syncers.switch is called by viewSpec:

https://github.com/specify/specify7/blob/3ffb5e7fa9e14b7421ff946bb9bac7cfd75ee274/specifyweb/frontend/js_src/lib/components/FormEditor/viewSpec.ts#L61-L89

switch in turn uses spread syntax to augment the passed in arguments:
https://github.com/specify/specify7/blob/3ffb5e7fa9e14b7421ff946bb9bac7cfd75ee274/specifyweb/frontend/js_src/lib/components/Syncer/syncers.ts#L661

in the process, that breaks the `table` object - it's properties are preserved in a new object, but all of the methods are lost.